packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```
@@ -12058,6 +12059,36 @@ describe('plugin-meetings', () => {
+                false, // isGranting: Stopping share
+                false, // isContent: Whiteboard
+                USER_IDS.REMOTE_A // Beneficiary ID: Remote user A
+              );
+
+              // Validate the payload changes and status updates
+              payloadTestHelper([data1]);
+
+              // Specific assertions for guest
+              assert.equal(meeting.shareStatus, SHARE_STATUS.REMOTE_SHARE_ACTIVE);
+            });
+          });
```

Biome (1.9.4)

[error] Excessive `describe()` nesting detected.

Excessive nesting of describe() calls can hinder test readability.
Consider refactoring and reduce the level of nested describe to improve code clarity.

(lint/complexity/noExcessiveNestedTestSuites)